src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.report;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {

```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line at the start of the class definition.